cel/options.go

```diff
@@ -29,6 +29,20 @@ import (
 	exprpb "google.golang.org/genproto/googleapis/api/expr/v1alpha1"
 )
 
+// These constants beginning with FEATURE_ enable optional behavior in
+// the library.  See the documentation for each constant to see its
+// effects, compatibility restrictions, and standard conformance.
+const (
+	_ = iota
```

Comment:
You could collapse these two lines, but I leave it to you to decide what's most readable here:

```
FeatureDisableDynamicAggregateLiterals = iota + 1
```

Reply:
I'll keep it as-is so the first definition looks like the rest, and so we can freely reorder them.